clientv3/integration/watch_keepalive_test.go

```diff
@@ -35,6 +35,7 @@ func TestWatchKeepAlive(t *testing.T) {
 	clus := integration.NewClusterV3(t, &integration.ClusterConfig{
 		Size:                 3,
 		GRPCKeepAliveMinTime: time.Millisecond, // avoid too_many_pings
+		ClientDebug:          true,
```

Comment:
not sure we should enable this by default for every CI run. probably should be enabled when we really debug stuff.